### ISSUE TITLE
Question about loss function

### ISSUE THREAD

**Issue by Geeks-Z** [Lee-JH-KR/l2p-pytorch#11]
In `engine.py` line 68 
`loss = loss - args.pull_constraint_coeff * output['reduce_sim']`
In the paper is
`loss = loss + args.pull_constraint_coeff * output['reduce_sim']`?

**Comment by prachigarg23** [comment]
Hi @Geeks-Z , I believe this is a typing error in the paper because the objective is to ' pull selected keys closer to corresponding query features' by maximising the cosine similarity of top_k selected keys. Hence, the minus sign in the code should be correct.
